src/CalculatorPage.jsx

```diff
@@ -0,0 +1,17 @@
+import Calculator from "./components/Calculator";
+
+export default function CalculatorPage() {
+  const header = {
+    fontSize: '1.5rem',
+    position: 'absolute',
+    top: '-7px',
+    left: '15px',
+  }
+
+  return (
+    <>
+      <h1 style={header}>Let us do some math!</h1>
+      <Calculator />
+    </>
```

Comment:
- [ ] The "Math Magicians" text-based logo should appear on all the pages just like in the wireframe. I suggest you set this up in your navigation so that you don't repeat code in your components 👍 
## Wireframe
![image](https://user-images.githubusercontent.com/46839548/235907152-b4839da4-91c9-4f0c-a8fe-786f08cfe52a.png)
## Your Design
![image](https://user-images.githubusercontent.com/46839548/235907325-927a7858-f8be-4487-b774-1c4ad549939a.png)